trackhubs/models.py

```diff
@@ -372,7 +373,10 @@ def generate_browser_links(self):
             },
         }
 
-        division = species_assemblies_divisions.get(species_scientific_name.lower()).get(assembly_name.lower())
+        try:
+            division = species_assemblies_divisions.get(species_scientific_name.lower()).get(assembly_name.lower())
+        except AttributeError:
+            division = None
```

Comment:
Another option here which uses fewer lines is to default the first `get` to an empty dict, ie
```
division = species_assemblies_divisions.get(species_scientific_name.lower(), {}).get(assembly_name.lower())
```
but using `AttributeError` works as well, I don't mind which you use.

Reply:
Can we keep the `try except`, I find it easier on the eye 😄